SWLOR.Game.Server/Feature/LootTableDefinition/ViscaraLootTableDefinition.cs

```diff
@@ -101,6 +104,19 @@ private void MandalorianRanger()
                 .AddItem("m_plexiplate", 4, 1, true);
         }
 
+        private void MandalorianCrate()
```

Comment:
Same comment here about separating creatures from resources. Can you move this to the Viscara resources class?